api/rds.proto

```diff
@@ -362,21 +438,36 @@ message Route {
 // endpoints that they wish to get “perfect” statistics on. Virtual cluster
 // statistics are perfect in the sense that they are emitted on the downstream
 // side such that they include network level failures.
+//
+// Examples:
+//
+// * The regex */rides/\d+* matches the path */rides/0*
+// * The regex */rides/\d+* matches the path */rides/123*
+// * The regex */rides/\d+* does not match the path */rides/123/456*
+//
+// Documentation for :ref:`virtual cluser statistics <config_http_filters_router_stats>`.
```

Comment:
Typo: cluster. I'd recommend just looking at this doc will spell check mode turned on in your editor to catch stuff I'm probably missing as well.